htmldocx/h2d.py

```diff
@@ -372,10 +378,21 @@ def ignore_nested_tables(self, tables_soup):
             new_tables.append(table)
             nest = len(table.find_all('table'))
         return new_tables
-    
+
+    def get_table_rows(self, table_soup):
+        # If there's a header, body, footer or direct child tr tags, add row dimensions from there
+        return table_soup.select(', '.join(self.table_row_selectors), recursive=False)
+
+    def get_table_columns(self, table_soup, row):

```

Comment:
Do we need to include `table_soup` here as a parameter?

Reply:
I don't think so. Looks like it's left over from the workflow. I'll remove:)

Reply:
@pqzx removed:)

Reply:
Thanks for the update :)